bot/internal/review/review.go

```diff
@@ -385,24 +374,12 @@ func (r *Assignments) getAdminReviewers(author string) []string {
 func (r *Assignments) getCodeReviewerSets(e *env.Environment) ([]string, []string) {
 	// Internal non-Core contributors get assigned from the admin reviewer set.
 	// Admins will review, triage, and re-assign.
-	v, ok := r.c.CodeReviewers[e.Author]
-	if !ok || v.Team == env.InternalTeam {
+	if !r.IsInternal(e.Author) {
```

Comment:
This change resolves the issue where a core engineer not defined in the cloud reviewer map submits a PR to the cloud repo but requires admin approval. Calling IsInternal now checks core, docs, **and cloud** maps to determine if the author is internal.